### PR TITLE
update cache Explorer to work with new devtools-exchange types

### PR DESCRIPTION
**Pull request by wgolledge**
## About

- Added new `DebugMessage` types for the cache Explorer

### CODE REVIEW

File: src/panel/context/Explorer/index.tsx [hunk 1]
```diff
@@ -30,17 +30,23 @@ export const ExplorerProvider: FC = ({ children }) => {
   >(undefined);
 
   useEffect(() => {
-    return addMessageHandler((o: DevtoolsExchangeOutgoingMessage) => {
-      if (o.type === "disconnect") {
+    return addMessageHandler((e: DevtoolsExchangeOutgoingMessage) => {
```
Comment (andyrichardson) [inline src/panel/context/Explorer/index.tsx]: Think might help differentiate _chrome messages_ from _debug events_

```suggestion
    return addMessageHandler((message: DevtoolsExchangeOutgoingMessage) => {
```

File: yarn.lock [hunk 2]
```diff
@@ -1865,6 +1865,11 @@ abab@^2.0.0:
   resolved "https://registry.yarnpkg.com/abab/-/abab-2.0.3.tgz#623e2075e02eb2d3f2475e49f99c91846467907a"
   integrity sha512-tsFzPpcttalNjFBCFMqsKYQcWxxen1pgJR56by//QwvJc4/OUS3kPOOttx2tSIfjsylB0pYu7f5D3K1RCxUnUg==
```
Comment (andyrichardson) [inline yarn.lock]: Any idea what caused the lock file changes?

File: src/panel/context/Explorer/index.tsx [hunk 3]
```diff
@@ -30,17 +30,23 @@ export const ExplorerProvider: FC = ({ children }) => {
...
+      if (e.type === "disconnect") {
         setOperations({});
         return;
       }
 
-      if (o.type === "response" && o.data.data) {
+      if (e.type !== "debug") {
```
Comment (andyrichardson) [inline src/panel/context/Explorer/index.tsx]: If we add this, we should be able to completely remove the second conditional on line 45

```suggestion
      if (e.type !== "debug" || e.data.type !== "update") {
```

I can't see a reason why `debugEvent.data` would be undefined so I think we're safe to omit that.

File: yarn.lock (hunk 2 above)
Comment (wgolledge) [reply yarn.lock]: Nope - they shouldn;t be here!

**Pull request merged**